core/src/main/java/io/questdb/cairo/wal/WalWriter.java

```diff
@@ -1177,7 +1173,12 @@ private void openNewSegment() {
     }
 
     private void releaseSegmentLock(int segmentId, int segmentLockFd) {
-        if (!ff.close(segmentLockFd)) {
+        if (ff.close(segmentLockFd)) {
+            LOG.info().$("released segment lock [walId=").$(walId)
```

Comment:
nit: can you move this to debug level please? There is already excessive logging about locks with little value. Debug logs are saved for test runs.

Reply:
I put this in because there is nothing to log lock release.

Reply:
All to debug then

Reply:
Alright, I'll update `TableUtils` then.